containers/python-3-pypy/README.md

```diff
@@ -0,0 +1,52 @@
+# PyPy (Community)
+
+## Summary
+
+*Develop python applications using the [PyPy](https://www.pypy.org/) interpreter*
```

Comment:
I don't believe links will render properly. Let's remove it from this spot since it drives VS Code UX.

```suggestion
*Develop python applications using the PyPy interpreter*
```